common/culture/innovations/wc_cultural_maa_innovations.txt

```diff
@@ -344,6 +370,22 @@ innovation_night_elf_aerial = {
 	flag = global_maa
 }
 
+innovation_nerubian_tactics = {
+	group = culture_group_regional
```

Comment:
I would give strong hill and mountain bonuses to nerubian units and arachnids since they're supposed to make caves there.